pytorch_lightning/utilities/data.py

```diff
@@ -197,6 +197,8 @@ def _update_dataloader(
             f" can be passed to its parent's `{argument}=...` `__init__` argument. This is likely caused by allowing"
             f" passing both a custom argument that will map to the `{argument}` argument as well as `**kwargs`."
             f" `kwargs` should be filtered to make sure they don't contain the `{argument}` key."
+            " This argument was automatically passed to your DataLoader by PyTorch Lightning,"
+            " however you are also passing it in your implementation."
```

Comment:
Don't these 2 new lines say the same as the message above?

Reply:
I don't think so - the message above does not say anything about the fact it is PL code that is injecting some kwargs there and might confuse people